.gitlab-ci.yml

```diff
@@ -0,0 +1,108 @@
+# see https://docs.gitlab.com/ce/ci/yaml/README.html for all available options
+
+variables:
+  SCHEDULER_PARAMETERS: "-J ArborX_CI -W 0:30 -nnodes 1 -P CSC333 -alloc_flags gpumps"
+
+stages:
+  - buildDependencies
+  - buildArborX
+  - runBenchmarks
+  - compare
+
+BuildBoost:
+  stage: buildDependencies
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - BOOST_VERSION=1.67.0 &&
+      BOOST_VERSION_UNDERSCORE=$(echo "$BOOST_VERSION" | sed -e "s/\./_/g") &&
+      BOOST_URL=https://dl.bintray.com/boostorg/release/${BOOST_VERSION}/source &&
+      BOOST_ARCHIVE=boost_${BOOST_VERSION_UNDERSCORE}.tar.bz2 &&
+      wget --quiet ${BOOST_URL}/${BOOST_ARCHIVE} &&
+      mkdir -p boost &&
+      tar -xf ${BOOST_ARCHIVE} -C boost --strip-components=1 &&
+      cd boost &&
+      ./bootstrap.sh --prefix=${CI_PROJECT_DIR}/boost.install &&
+      ./b2 -j8 hardcode-dll-paths=true dll-path=${CI_PROJECT_DIR}/boost.install link=static variant=release cxxflags=-w install
+  tags:
+    - nobatch
+  artifacts:
+    paths:
+      - boost.install/
+
+BuildBenchmark:
+  stage: buildDependencies
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - git clone https://github.com/google/benchmark.git -b v1.4.1 &&
+      cd benchmark &&
+      git clone https://github.com/google/googletest.git -b release-1.8.1 &&
+      mkdir build && cd build &&
+      cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=${CI_PROJECT_DIR}/benchmark.install .. &&
+      make && make install
+  tags:
+    - nobatch
+  artifacts:
+    paths:
+      - benchmark.install/
+
+BuildKokkos:
+  stage: buildDependencies
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - git clone --depth=1 --branch 3.1.00 https://github.com/kokkos/kokkos.git &&
+      cd kokkos &&
+      mkdir build &&
+      cd build &&
+      cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${CI_PROJECT_DIR}/kokkos.install -DCMAKE_CXX_COMPILER=${CI_PROJECT_DIR}/kokkos/bin/nvcc_wrapper -DCMAKE_CXX_EXTENSIONS=OFF -DKokkos_ENABLE_SERIAL=ON -DKokkos_ENABLE_OPENMP=ON -DKokkos_ENABLE_CUDA=ON -DKokkos_ENABLE_CUDA_LAMBDA=ON -DKokkos_ARCH_POWER9=ON -DKokkos_ARCH_VOLTA70=ON .. &&
+      make &&
+      make install
+  tags:
+    - nobatch
+  artifacts:
+    paths:
+      - kokkos.install/
+
+BuildArborX:
+  stage: buildArborX
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - mkdir build && cd build &&
+      cmake -DCMAKE_PREFIX_PATH="${CI_PROJECT_DIR}/kokkos.install;${CI_PROJECT_DIR}/benchmark.install;${CI_PROJECT_DIR}/boost.install" -DARBORX_ENABLE_MPI=ON -DARBORX_ENABLE_TESTS=OFF -DARBORX_ENABLE_BENCHMARKS=ON -DCMAKE_CXX_COMPILER=${CI_PROJECT_DIR}/kokkos.install/bin/nvcc_wrapper .. &&
+      make VERBOSE=1 ArborX_BoundingVolumeHierarchyRegression.exe
```

Comment:
I assume `VERBOSE=1` is simply a leftover from when you were debugging? There is another one below.